### PR TITLE
Core

### PR DESCRIPTION
**Pull request by SooLee**
Use foursight_core

### CODE REVIEW

**Review by willronchetti — COMMENTED** [review]
I have some small comments I've made on `pyproject.toml`, other than those if this is deploying/running then I think these changes should be good. I can make other comments on aspects of the refactor into foursight-core but since that seems to be done for the most part and working I"ll hold off on that for now, we can always make incremental improvements in the future.

File: pyproject.toml [hunk 1]
```diff
@@ -11,7 +11,7 @@ packages = [
 [tool.poetry.dependencies]
 python = ">=3.6,<3.7"
 dcicutils = "1.1.0b1"
-click = "6.7"
+click = ">=6.7"
```
[the remark this answers is not in the document]
Comment (SooLee) [reply pyproject.toml:14]: ok I will do that. The actual version installed is 7.1.2, so I will make it "^7.1.2"

File: pyproject.toml [hunk 2]
```diff
@@ -23,9 +23,14 @@ elasticsearch-dsl = "6.4.0"
 python-Levenshtein = "0.12.0"
 gitpython = "^3.1.2"
 pytz = "^2020.1"
+## adding foursight-core
+# use below for deployment
+foursight-core = "0.0.1b47"
```
[the remark this answers is not in the document]
Comment (SooLee) [reply pyproject.toml:28]: yup

File: pyproject.toml [hunk 3]
```diff
@@ -23,9 +23,14 @@ elasticsearch-dsl = "6.4.0"
...
+# use below for tests but not for deployment
+#foursight-core = { git = "https://github.com/4dn-dcic/foursight-core.git", branch="master" }
 
 [tool.poetry.dev-dependencies]
-chalice = "1.11.1"
+chalice = ">=1.21.0"
```
[the remark this answers is not in the document]
Comment (SooLee) [reply pyproject.toml:33]: sure

**Pull request merged**